geometry_msgs/PoseStamped start_pose
---
geometry_msgs/PoseStamped next_frontier
